src/components/InfoPanel.component.js

```diff
@@ -137,6 +171,25 @@ class InfoPanel extends Component {
           </div>
         )}
 
+        {!!media.length &&
+          isEditing &&
+          isRealTimePOI && (
+            <FieldGroup
+              inputType="text"
+              value={captions}
+              placeholder="Change Media Caption Here"
+              onChange={captions =>
```

Comment:
what you're calling `captions` here is actually the `event` object. you should call it `event` or `e`